.buildkite/scripts/integration_tests.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
These files should have a License header ? 
If so, how can it be added ? using golicenser?

Reply:
Yeah, I think they can be kept like this.